### PR TITLE
Add software install guide markdown document and edit root readme 

### PR DESCRIPTION
**Pull request by aidandriscoll**
Please check document for correctness. Please feel free to add images if you feel it is appropriate. 

Note: The install guide does not include any information on installing CLI git from git-scm, as it is bundled with thee GitHub Desktop installer and does not need to be installed separately. Source: https://github.com/git-guides/install-git

### CODE REVIEW

File: docs/software_guide.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+3. <b>Prepare IntelliJ IDEA for Android Development</b>
+    1. Click `Create New Project`
+    2. Install the Java JDK
+        At the top of the window you should see a drop down for `Project SDK`, click it, then click `Download JDK...` select version 11 and set the vendor to `Eclipse Temurin (AdoptOpenJDK HotSpot)` then click download. 
+    3. Install Android SDK
+        On the side bar, click Android, then when it mentions that you do not have the Android SDK, follow it's prompts to install it. 
+
+
+4. <b>Clone the FTC-2021 repository</b>
```
Comment (Kappabyte) [inline docs/software_guide.md:37]: If people are not a part of the organization yet, they will not see the repo in Github Desktop. Maybe add a step that gets them to add us to the org first.

**Review by Kappabyte — CHANGES_REQUESTED** [review]
Looks good. Just a few minor things that should probably be changed.

**Comment by aidandriscoll** [comment]
Implemented the suggested changes into PR#3, please review.

File: docs/software_guide.md [hunk 2]
```diff
@@ -20,47 +21,63 @@ If you do have a ChromeBook and can not bring in a Windows, MacOS, or Linux lapt
...
 
     In GitHub Desktop, after you have signed in, click current repository, then add, then clone, and find FTC-2021 in the list, hit clone. After, hit fetch orgin at the top of the window. 
 
 
-5. <b>Open the repository in IntelliJ IDEA</b>
 
-    On the main page of GitHub Desktop, there is an option to `Open the repository in your external editor`, if it says `Open in JetBrains IntelliJ Idea`, please click that button.
+6. <b>Open the repository in IntelliJ IDEA</b>
+
+    On the main page of GitHub Desktop, there is an option to `Open the repository in your external editor`, if it says `Open in JetBrains IntelliJ Idea`, please click that button. If it does not show up, try closing and reopening GitHub desktop.
```
Comment (Kappabyte) [inline docs/software_guide.md:56]: I would remove this section. People should just open it directly in Intellij.

**Review by Kappabyte — CHANGES_REQUESTED** [review]
Looks good. Just change the one thing, than I'll merge. Also, please add .DS_Store to the .gitignore

File: docs/software_guide.md (hunk 2 above)
Comment (Kappabyte) [reply docs/software_guide.md:56]: What I'm trying to say, is this part should at least come after the open directly in intellij section if at all. It should be clearly marked as optional.

**Comment by aidandriscoll** [comment]
Remove the Open in GitHub desktop section.

**Review by Kappabyte — APPROVED** [review]
Looks Good!

**Pull request merged**